src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -101,6 +102,22 @@ public Response getCollectionById(
     }
   }
 
+  @DELETE
+  @Path("{collectionId}")
+  @Produces("application/json")
+  @RolesAllowed({ADMIN, RESEARCHER})
+  public Response deleteDarCollrection(@Auth AuthUser authUser, @PathParam("collectionId") Integer collectionId) {
+    try {
+      User user = userService.findUserByEmail(authUser.getEmail());
+
+      darCollectionService.deleteByCollectionId(user, collectionId);
+
+      return Response.ok().build();
```

Comment:
Super minor nit, remove the extra white space:
```suggestion
      User user = userService.findUserByEmail(authUser.getEmail());
      darCollectionService.deleteByCollectionId(user, collectionId);
      return Response.ok().build();
```